StyleGuide.md

```diff
@@ -0,0 +1,372 @@
+# Style Guide
+
+> A programmer does not primarily write code; rather, he primarily writes to
+> another programmer about his problem solution. The understanding of this
+> fact is the final step in his maturation as technician.
+>
+> — [What a Programmer Does, 1967][1]
+
+This style guide consists of two sections:
+  * Linting - with guidelines about code content that are considered good
```

Comment:
About Linting.
I have to first ask.
What is the intention of this style guide.
  - Is the style guide of how to write code in erlfmt?
  - Or?

Reply:
I updated this section to the following. What do you think?
> This style guide represents guidelines for writing good Erlang code according
> to the authors of erlfmt. It consists of two sections:
>   * Linting - with guidelines about code content that are considered good
>     practice and need to be applied manually
>   * Formatting - with guidelines about code layout that can be applied
>     automatically by using `erlfmt`

Reply:
Sounds good 👍